felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1568,10 +1585,11 @@ func (m *bpfEndpointManager) ensureBPFDevices() error {
 		return nil
 	}
 
-	nl, err := netlinkshim.NewRealNetlink()
+	nl, err := netlink.NewHandle(syscall.NETLINK_ROUTE)
```

Comment:
If not shimming this, can you just use netlink's package-level API:
```
netlink.LinkByName()
```
Instead of 
```
Handle.LinkByName()
```
Netlink maintains a package level shared handle.